core/src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/AwsConnector.java

```diff
@@ -530,7 +516,7 @@ private void resumeAutoScalingPolicies(Stack stack, AwsCredential awsCredential)
                 }
             } catch (AmazonServiceException e) {
                 if (e.getErrorMessage().matches("Resource.*does not exist for stack.*")) {
-                    LOGGER.info(e.getErrorMessage());
+                    LOGGER.info("Resource doesn't exist for stack: ", e);
```

Comment:
I'm not sure if this is a real error here or not (we don't do anything with it other than logging). If not then I don't think the stack trace should be logged, if yes then we should do something with it.